source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+.. _clear-linux-dissector:
+
+Clear Linux\* Dissector
+#######################
+
+The Clear Linux\* Dissector tool compares contents of Linux-based images. It
+supports the comparison of |CL| and Yocto Project\* images.
+
+.. contents:: :local:
+   :depth: 1
+
+Description
+***********
+
+Clear Linux Dissector provides tools for comparing |CL| images and Yocto Project
+images.
+
+* **View distribution data.**
+
+  View the distribution data of a specific |CL| and Yocto Project distribution
+  side-by-side. Helpful to determine if a |CL| version contains the packages and
+  patches needed to move from a specific Yocto Project distribution to |CL|.
+
+* **Compare images.**
+
+  Compare |CL| and Yocto Project images to identify which packages you will need
+  to move from a Yocto Project image to a |CL| image.
+
+* **Compare releases.**
+
+  Compare different |CL| releases to see changes between |CL| versions.
+
+Prerequisites
+*************
+
+#. Must be on a Linux machine.
+
+Set Up Clear Linux Dissector
+****************************
+
+Clear Linux Dissector is a web application, which means you will need to set up
+an instance of the application in order to use it.
+
+#. First, clone the clear-linux-dissector-web git repository.
+
+   .. code-block:: bash
+
+      git clone https://github.com/intel/clear-linux-dissector-web.git
+
+#. Change directory into clear-linux-dissector-web.
+
+   .. code-block:: bash
+
+      cd clear-linux-dissector-web
+
+#. Run the dockersetup.py script.
+
+   If you are behind a proxy, you must set the command line variables
+   :command:`--http-proxy`, :command:`--https-proxy`, and :command:`--socks-proxy`.
```

Comment:
These are command-line options rather than variables.

You may need --no-proxy as well if you want to access internal URLs which the proxy doesn't handle; though I suspect the use cases for this (derivative import) aren't going to be covered in this doc.